SpatialMapping/PlaneFinding/UnityAddon/package.json

```
@@ -1,8 +1,9 @@
 {
     "name": "com.microsoft.mixedreality.toolkit.planefinding",
     "displayName": "Mixed Reality Toolkit Plane Finding",
-    "version": "1.0.0-preview.1",
+    "version": "1.0.0",
```

```suggestion
    "version": "1.0.2",
```

To match the change log

---

Oh whoops, the changelog change you're looking at is actually for GPU Stats. Meant to break it out into a different PR, but it's in here...